src/acceptance-test/java/uk/gov/ida/verifyserviceprovider/NonMatchingEidasAcceptanceTest.java

```diff
@@ -28,7 +43,17 @@
     @ClassRule
     public static NonMatchingVerifyServiceProviderAppRule appWithEidasDisabled = new NonMatchingVerifyServiceProviderAppRule(false);
 
-     @Test
+    private OpenSamlXmlObjectFactory openSamlXmlObjectFactory;
+
+    @Before
+    public void setUp() throws Exception {
+        openSamlXmlObjectFactory = new OpenSamlXmlObjectFactory();
+    }
+
+    @Before
```

Comment:
This @Before is floating around on its own and looks like it shouldn't be there at all.